utils/providerAPI/getUserRepositories.ts

```diff
@@ -7,15 +7,10 @@ import AuthInfo from "../../types/AuthInfo";
 import { bitbucketAccessToken } from "./auth";
 
 type GithubRepoObj = {
-	name: string,
-	full_name: string,
-	created_at: string,
-	updated_at: string,
-	license: any,
-	private: boolean,
-	owner: object,
-	html_url: string,
-	[property: string]: any
+	name: string;
+	owner: {
+		login: string;
+	};
```

Comment:
I think we should keep the `private` boolean here. We will use that very soon to display which repositories are free for use and which will cost them money.

Reply:
I agree with @avikalpg 's point